linchpin/defaults/schemas/schema_v4.json

```diff
@@ -108,8 +108,35 @@
                                  "keyvalue": {
                                      "type": "array"
                                  },
-                                 "hostrequire": {
-                                     "type": "array"
+                                 "hostrequires": {
+                                     "type": "array",
+                                     "items": {
+                                         "not": {
+                                             "allOf": [
+                                                 {"required": ["force", "tag"]}
+                                             ]
+                                         },
```

Comment:
We have written a schemacheck.yml, included at the same level as site.yml, which you could use to check your schema. You use it something like this: 

```bash
ansible-playbook -vvvv linchpin/provision/schemacheck.yml -e \ 
"topology='/home/herlo/Projects/linchpin/linchpin/examples/topologies/dummy_cluster.yml'" \ 
-e 'schema=../defaults/schemas/schema_v4.json'
```